src/commands/uploadMobile.js

```diff
@@ -0,0 +1,73 @@
+import { uploadMachO } from '../uploadMacho.js';
+import { uploadProguard } from '../uploadProguard.js';
+
+const PLATFORMS = {
+  IOS: 'ios',
+  ANDROID: 'android',
+};
+
+const PLATFORM_UPLOAD = {
+  [PLATFORMS.IOS]: uploadMachO,
+  [PLATFORMS.ANDROID]: uploadProguard,
+};
+
+export const command = 'upload-mobile <path(s..)>';
+export const describe = 'Upload debug files for a mobile release';
+
+const sampleiOSUploadPath = '/build/MyApp-iphoneos.xcarchive/dSYMs/MyApp.framework.dSYM/Contents/Resources/';
+const sampleAndroidUploadPath = '/app/build/outputs/mapping/debug/mapping.txt';
+const usageStr = `Usage: logrocket upload-mobile -r <release> -pf <platform> <path(s..)>
+For android, include one path directly to debug file, like \`${sampleAndroidUploadPath}\`
+See android dev docs for information on how to shrink and obfuscate your code https://developer.android.com/build/shrink-code#enable
+`;
+const demandPathStr = `Missing upload path, e.g.
+logrocket upload-mobile -pf ios -r 1.2.3 ${sampleiOSUploadPath}
+-OR-
+logrocket upload-mobile -pf android -r 1.2.3 ${sampleAndroidUploadPath}
+`;
+
+export const builder = (args) => {
+  args
+    .usage(usageStr)
+    .option('r', {
+      alias: 'release',
+      type: 'string',
+      describe: 'The release version for these files',
+      demand: 'You must specify a release, use -r or --release',
+    })
+    .demand(1, demandPathStr)
+    .option('pf', {
```

Comment:
Pretty minor, but it looks like when you have a multi-letter flag like this the formatter assumes it's intended as an expanded param with two dashes (help text says `--pf, --platform`) - having this as `-p` feels more conventional to me - any objections to that?

Reply:
only that `-p` is used for `urlPrefix` in our existing upload command (so users might expect to use it that same way here), but as long as they look at the parameters it should be fine

Reply:
Ooh good point; guess neither is 100% intuitive but -p makes the most sense to me, unless you can think of a good synonym for 'platform' that would feel natural and starts with a different letter?